modules/lightning_features/lightning_media/src/Plugin/EntityBrowser/Widget/FileUpload.php

```diff
@@ -65,9 +65,12 @@ public function getForm(array &$original_form, FormStateInterface $form_state, a
     // it as the first validator, allowing it to accept only file extensions
     // associated with existing media bundles.
     if (empty($validators['file_validate_extensions'])) {
+      $entity_browser_info = $form_state->get('entity_browser');
```

Comment:
@phenaproxima To my knowledge, `FormStateInterface::get()` only accepts a single argument - `$property`. See: https://api.drupal.org/api/drupal/core%21lib%21Drupal%21Core%21Form%21FormStateInterface.php/function/FormStateInterface%3A%3Aget/8.3.x